### PR TITLE
ref(integrations): Add new feature flags

### PR DESCRIPTION
**Pull request by MeredithAnya**
Other half of https://github.com/getsentry/getsentry/pull/4132

We use the categories to feature gate in addition to using them as ways for users to filter what integrations they might be looking for in the Integration Directory. 

These were not previously turned into feature flags because they weren't used on paid plans. Now they are.

### CODE REVIEW

File: src/sentry/conf/server.py [hunk 1]
```diff
@@ -855,6 +855,13 @@ def create_partitioned_queues(name):
     "organizations:integrations-issue-sync": True,
     # Enable interface functionality to receive event hooks.
     "organizations:integrations-event-hooks": False,
+    # Enable integration functionality to work with alert rules
+    "organizations:integrations-alert-rule": False,
```
Comment (scefali) [inline src/sentry/conf/server.py:859]: @MeredithAnya Shouldn't the default value be `True` here so on-prem folks get these features by default? I realize `organizations:integrations-event-hooks` probably should be updated to `True` as well.

File: src/sentry/integrations/pagerduty/integration.py [hunk 2]
```diff
@@ -134,7 +134,7 @@ class PagerDutyIntegrationProvider(IntegrationProvider):
     key = "pagerduty"
     name = "PagerDuty"
     metadata = metadata
-    features = frozenset([IntegrationFeatures.ALERT_RULE])
+    features = frozenset([IntegrationFeatures.ALERT_RULE, IntegrationFeatures.INCIDENT_MANAGEMENT])
```
Comment (scefali) [inline src/sentry/integrations/pagerduty/integration.py:137]: @MeredithAnya The feature descriptions should be updated as well to match

**Review by scefali — CHANGES_REQUESTED** [review]

File: src/sentry/integrations/pagerduty/integration.py (hunk 2 above)
Comment (MeredithAnya) [reply src/sentry/integrations/pagerduty/integration.py:137]: PagerDuty already had the feature description for incident management

**Review by scefali — APPROVED** [review]

File: src/sentry/conf/server.py [hunk 3]
```diff
@@ -854,7 +854,14 @@ def create_partitioned_queues(name):
...
     # Enable interface functionality to receive event hooks.
-    "organizations:integrations-event-hooks": False,
+    "organizations:integrations-event-hooks": True,
+    # Enable integration functionality to work with alert rules
+    "organizations:integrations-alert-rule": True,
+    # Enable integration functionality to work with alert rules (specifically chat integrations)
+    "organizations:integrations-chat-unfurl": True,
+    # Enable integration functionality to work with alert rules (specifically indicdent)
+    # management integrations)
+    "organizations:integrations-incident-management": True,
```
Comment (EvanPurkhiser) [inline src/sentry/conf/server.py:864]: Were these already added in `features/__init__`?

Comment (MeredithAnya) [reply src/sentry/conf/server.py:864]: nope - good find!

**Pull request merged**